### PR TITLE
nemesis for TiJepsen

### PR DESCRIPTION
**Pull request by UncP**
Signed-off-by: UncP <uncp.xu@gmail.com>

### CODE REVIEW

File: tidb/run [hunk 1]
```diff
@@ -0,0 +1,6 @@
+lein run test \
```
Comment (siddontang) [inline tidb/run]: run.sh 

Btw, I prefer moving this to a scirpts directory and we can provide multi other test scripts for different cases.

Comment (UncP) [inline tidb/run]: one script is enough, it's modifiable, we can choose which test to run and which nemesis to use, etc.

Comment (siddontang) [inline tidb/run]: 1. If we want to let SRE team run multi cases automatically, how do they do? 
2. If I want to run some other cases, how can I know the real name? Review the codes and find the correct one? 

The script is too simple.

Comment (UncP) [inline tidb/run]: ok

Comment (siddontang) [inline tidb/run]: IMO, you can rename run to run.sh first, then refactor the running scripts later.

File: tidb/run.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#! /bin/sh
```
Comment (siddontang) [inline tidb/run.sh]: remove this and add these to the ReadME

Comment (UncP) [inline tidb/run.sh]: ok

**Pull request merged**